unified-runtime/source/adapters/opencl/device.cpp

```diff
@@ -1753,9 +1753,16 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetGlobalTimestamps(
   oclv::OpenCLVersion DevVer, PlatVer;
   cl_device_id DeviceId = hDevice->CLDevice;
 
+  std::map<cl_int, ur_result_t> CustomErrMap{
+    {CL_INVALID_OPERATION, UR_RESULT_ERROR_UNSUPPORTED_FEATURE}
+  };
+
   // TODO: Cache OpenCL version for each device and platform
   auto RetErr = hDevice->getDeviceVersion(DevVer);
-  CL_RETURN_ON_FAILURE(RetErr);
+
```

Comment:
Why not just use `if (RetErr == CL_INVALID_OPERATION) return UR_RESULT_ERROR_UNSUPPORTED_FEATURE;`? It would avoid all this infrastructure.

Reply:
Sure, fixed